src/main/java/io/curity/example/democlient/config/KeyStoreConfig.java

```diff
@@ -0,0 +1,44 @@
+package io.curity.example.democlient.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "client.authentication.asymmetric-key")
+public class KeyStoreConfig {
```

Comment:
This could be a Java `record`:

```java
public record KeyStoreConfig(
        String keyStoreFileName,
        String keyStorePassword,
        String keyStoreAlias,
        String keyStoreType
) {}
```